homeassistant/components/nibe_heatpump/utils.py

```diff
@@ -0,0 +1,43 @@
+"""The Nibe Heat Pump utils."""
+
+from __future__ import annotations
+
+import asyncio
+
+from . import LOGGER
+
+
```

Comment:
Can the retry code live in the library?

Reply:
Don't think it make sense there. But i could probably add a dependency on: https://github.com/jd/tenacity maybe we even want to have as a core dependancy

Reply:
Switch to tenacity package instead of homegrown solution